samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -343,15 +343,21 @@ mfxStatus sTask::Reset()
 }
 
 mfxStatus CEncodingPipeline::AllocateExtMVCBuffers()
```

Comment:
Can we fully delete this function? It's used in one place only.

Reply:
[CRegionEncodingPipeline::Init()](https://github.com/Intel-Media-SDK/MediaSDK/blob/c22544bcf12de3f7c4ab00276d0fb604fea5892b/samples/sample_encode/src/pipeline_region_encode.cpp#L420) is another place. Not that one is very useful.

Reply:
Why another? As far as I see this is the only place where it's called. I propose to remove

Reply:
@dmitryermilov 1st place is in [CEncodingPipeline::Init()](https://github.com/Intel-Media-SDK/MediaSDK/blob/c22544bcf12de3f7c4ab00276d0fb604fea5892b/samples/sample_encode/src/pipeline_encode.cpp#L1829). Sure, let's remove.